src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -380,9 +385,21 @@ namespace Microsoft.FSharp.Core
     /// <summary>Represents a out-argument managed pointer in F# code. This type should only be used with F# 4.5+.</summary>
     type outref<'T> = byref<'T, ByRefKinds.Out>
 
+    module PrimReflectionAdapters =
+
+        type System.Type with
+            
+            // Note, this differs slightly from the nettstandard1.6 FX_RESHAPED_REFLECTION case as it looks for non-public methods.
```

Comment:
Not sure if this helps, but all that stuff is now removed since FSharp.Core is NS2.0 only now.

Reply:
Thanks, I've removed the comment